common/src/main/java/com/revenuecat/purchases/common/HTTPClient.kt

```diff
@@ -177,7 +177,7 @@ class HTTPClient(
         val verificationResult = if (shouldSignResponse &&
             RCHTTPStatusCodes.isSuccessful(responseCode)
         ) {
-            verifyResponse(path, connection, payload, nonce)
+            verifyResponse(urlPathWithVersion, connection, payload, nonce)
```

Comment:
This was previously only used for logging, so it's fine to change it to include the api version (`/v1`)